plugin/storage/es/options.go

```diff
@@ -83,35 +83,41 @@ type namespaceConfig struct {
 // NewOptions creates a new Options struct.
 func NewOptions(primaryNamespace string, otherNamespaces ...string) *Options {
 	// TODO all default values should be defined via cobra flags
+	defaultConfig := config.Configuration{
+		Username:          "",
+		Password:          "",
+		Sniffer:           false,
+		MaxSpanAge:        72 * time.Hour,
+		NumShards:         5,
+		NumReplicas:       1,
+		BulkSize:          5 * 1000 * 1000,
+		BulkWorkers:       1,
+		BulkActions:       1000,
+		BulkFlushInterval: time.Millisecond * 200,
+		Tags: config.TagsAsFields{
+			DotReplacement: "@",
+		},
+		Enabled:              true,
+		CreateIndexTemplates: true,
+		Version:              0,
+		Servers:              []string{defaultServerURL},
+		MaxDocCount:          defaultMaxDocCount,
+	}
 	options := &Options{
 		Primary: namespaceConfig{
-			Configuration: config.Configuration{
-				Username:          "",
-				Password:          "",
-				Sniffer:           false,
-				MaxSpanAge:        72 * time.Hour,
-				NumShards:         5,
-				NumReplicas:       1,
-				BulkSize:          5 * 1000 * 1000,
-				BulkWorkers:       1,
-				BulkActions:       1000,
-				BulkFlushInterval: time.Millisecond * 200,
-				Tags: config.TagsAsFields{
-					DotReplacement: "@",
-				},
-				Enabled:              true,
-				CreateIndexTemplates: true,
-				Version:              0,
-				Servers:              []string{defaultServerURL},
-				MaxDocCount:          defaultMaxDocCount,
-			},
-			namespace: primaryNamespace,
+			Configuration: defaultConfig,
+			namespace:     primaryNamespace,
 		},
 		others: make(map[string]*namespaceConfig, len(otherNamespaces)),
 	}
 
+	// Other namespaces need to be explicitly enabled.
+	defaultConfig.Enabled = false
```

Comment:
This should fix both the failing ES integration tests and Crossdock builds.

The cause was due to archive mode being enabled by default and attempting to either ping a non-existent ES for a version (failing ES integration test) or attempting to communicate with ES on the wrong (default) IP address (failing Crossdock builds).

The original behaviour was to have archive mode disabled.